flexmeasures/data/schemas/users.py

```diff
@@ -24,3 +24,4 @@ def validate_timezone(self, timezone):
     active = ma.auto_field()
     timezone = ma.auto_field()
     flexmeasures_roles = ma.auto_field()
+    last_login_at = ma.auto_field()
```

Comment:
If we expect a timezone-aware iso-formatted datetime, why use auto_field over our AwareDatetimeField (or similarly named) from our schemas folder?